tests/ha/test_server_pod_failure.py

```diff
@@ -757,9 +743,9 @@ def test_writes_deletes_during_server_pod_down(self):
         new_s3data = dict()
         for bkt in remain_bkts:
             new_s3data[bkt] = s3_data[bkt]
-        args = {'test_prefix': self.test_prefix, 'test_dir_path': self.test_dir_path,
-                'skipput': True, 'skipdel': True, 's3_data': new_s3data, 'di_check': True,
-                'output': rd_output}
+        args = {'test_prefix': test_del_prefix, 'test_dir_path': self.test_dir_path,
+                'skipput': True, 'skipdel': True, 's3_data': new_s3data, 'bkt_list': remain_bkts,
```

Comment:
why do we need bucket list to be passed here?

Reply:
remain_bkts = list(set(written_bck) - set(get_random_buck)), is the list of bucket which is specifically written for DELETEs operations, and if we go without passing list it will fetch all the list of bucket for the s3_test_obj, which will include bucketes writen specific for the WRITEs.